kratos/includes/mesh.h

```diff
@@ -271,13 +271,29 @@ class Mesh : public DataValueContainer, public Flags
     /** Returns the Node::Pointer  corresponding to it's identifier */
     typename NodeType::Pointer pGetNode(IndexType NodeId)
     {
-        return (*mpNodes)(NodeId);
+        KRATOS_TRY
```

Comment:
I would avoid the KRATOS_TRY and catch in this level for two reason:
- KRATOS_ERROR already stores this method as error source
- It is not likely to have an error from downstream so the overhead of try and catch is less justified. (Nevertheless in this function the overhead is not significance because the function has many operations)